content/sensu-go/5.16/installation/install-sensu.md

```diff
@@ -77,7 +77,19 @@ sudo yum install sensu-go-backend
 
 {{< /language-toggle >}}
 
-##### 2. Configure and start
+##### 2. Initialize
+
+Run `sensu-backend init` to set up your Sensu administrator username and password. In this initialization step, you only need to provide a username and password string &mdash; no need for role-based access control (RBAC) or etcd.
+
+Replace `YOUR_USERNAME` and `YOUR_PASSWORD` with the username and password you want to use:
+
+{{< highlight shell >}}
+sensu-backend init --admin-username YOUR_USERNAME --admin-password YOUR_PASSWORD
+{{< /highlight >}}
```

Comment:
While this is a supported mechanism of the tool, it is also insecure. The tool's parameters will be visible to other users in the system.

A better approach is to set environment variables.

```
{{< highlight shell >}}
export SENSU_BACKEND_CLUSTER_ADMIN_USERNAME=YOUR_USERNAME
export SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD=YOUR_PASSWORD
sensu-backend init
{{< /highlight >}}